File: python/docs_code_snippets/knowledge-create.py
import time
from opperai import Opper

opper = Opper()

kb_name = f"docs-snippet-{int(time.time())}"

# --- docs ---
kb = opper.knowledge.create(name=kb_name)
print(f"Created: {kb.id} ({kb.name})")
# --- /docs ---

# Cleanup
opper.knowledge.delete(kb.id)

File: typescript/docs_code_snippets/knowledge-create.ts
import { Opper } from "opperai";

const opper = new Opper();

const kb = await opper.knowledge.create({ name: `docs-snippet-kb-${Date.now()}` });
console.log(`Created: ${kb.id} (${kb.name})`);

// Cleanup
await opper.knowledge.delete(kb.id);
